src/nunavut/lang/c/templates/_deserialize.j2

```diff
@@ -0,0 +1,170 @@
+{%- macro nunavut_get(bit_length=None) -%}
+{%- if bit_length is None -%}
+in_buffer, buf_size_bytes, offset
+{%- else -%}
+in_buffer, buf_size_bytes, offset, {{ bit_length }}
+{%- endif -%}
+{%- endmacro %}
+
+{%- macro nunavut_assign(ref, name) -%}
+{%- if name == '' -%}
+{{ ref }}
+{%- else -%}
+{{ ref }}{{ name | id }}
+{%- endif -%}
+{%- endmacro %}
+
+
+{%- macro deserialize(type) -%}
+    {{ _deserialize_composite(type, 'out_instance->', 0) }}
```

Comment:
You have a mix of using `out_instance` as a parameter, as you do here, and as a constant, as you do in the _deserialize_array macro. Please pick a single strategy and use it consistently.